src/collections/collections.md

```diff
@@ -0,0 +1,258 @@
+# Collections
```

Comment:
From somebody who isn't as experienced as you are:
Great read! For me all key information is there.